content/plugins/aws.md

```diff
@@ -65,8 +88,10 @@ All cloud resource nodes have common properties:
       * If the server is to have an agent installed on it, it should use the agents security group. If you are using a manager bootstrapped with the standard aws-manager-blueprint, there is a provider context dictionary on the manager that provides this value to the plugin. You can also use other security groups by:
         * `security_groups`: list of security group names.
         * `security_group_ids`: a list of security group IDs.
-  * `image_id` The AMI image id for the instance. For more information, please refer to: [http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/finding-an-ami.html](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/finding-an-ami.html)
-  * `instance_type` The instance type for the instance. For more information, please refer to: [http://aws.amazon.com/ec2/instance-types/](http://aws.amazon.com/ec2/instance-types/)
+      * If you want to specify the availability_zone for your instance, you must use the "placement" key.
+  * `image_id` The AMI image id for the instance. For more information, please refer [here](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/finding-an-ami.html).
+  * `instance_type` The instance type for the instance. For more information, please refer [here](http://aws.amazon.com/ec2/instance-types/).
+  * `name` Allows you to provide a name to your instance. It will be tagged with this name and will show up in your AWS console.
```

Comment:
maybe each of these should have an - (e.g. EXAMPLE) so that users can just copy and try?